src/api/types.ts

```diff
@@ -3,3 +3,15 @@ export interface CreateWalletArguments {
   n: number;
   m: number;
 }
+
+export interface CreatePSBTOutput {
+  address: string | undefined;
+  btc: number | undefined;
+}
+
+export interface CreatePSBTOutputs extends Array<CreatePSBTOutput>{}
```

Comment:
This is a little unnecessary, you can just shorthand an array of anything as `type[]`, so `CreatePSBTOutput[]` is pretty trivial to do.